src/Angor/Client/Pages/Invest.razor

```diff
@@ -434,44 +459,52 @@
 
     private async Task Send()
     {
-        var operationResult = await notificationComponent.LongOperation(async () =>
+        investSpinner = true;
+        StateHasChanged();
+        await Task.Delay(10);
+
+        try
         {
-            showCreateModal = false;
-            
             Debug.Assert(project != null, nameof(project) + " != null");
-            
-            project = new InvestorProject { ProjectInfo = project.ProjectInfo, Metadata = project.Metadata, SignedTransactionHex = signedTransaction!.ToHex() };
 
-            var investorProject = (InvestorProject)project;
-            
-            // remove signatures when requesting founder to sign 
-            investorProject.SignaturesInfo = new()
+            project = new InvestorProject
             {
-                ProjectIdentifier = investorProject!.ProjectInfo.ProjectIdentifier,
+                ProjectInfo = project.ProjectInfo, 
+                Metadata = project.Metadata, 
+                SignedTransactionHex = signedTransaction!.ToHex(), 
+                CreationTransactionId = project.CreationTransactionId
```

Comment:
Yes this is correct because we are creating the new `InvestorProject` type, the `CreationTransactionId` exists on the `Project` type loaded from the session like this
`project = SessionStorage.GetProjectById(ProjectId);`